pylib/iemweb/autoplot/scripts/p84.py

```
@@ -360,6 +370,14 @@ def set_data(ctx):
             clevs = pretty_bins(0, maxval)
         clevs[0] = 0.01
 
+    if ctx.get("rng") is not None:
+        try:
+            rng = [float(x) for x in ctx["rng"].split()]
+            if len(rng) == 2:
+                clevs = pretty_bins(rng[0], rng[1])
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)